src/main/java/ch/wisv/areafiftylan/extras/rfid/service/RFIDLinkRepository.java

```diff
@@ -19,13 +19,19 @@
 
 import ch.wisv.areafiftylan.extras.rfid.model.RFIDLink;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
+import java.util.Collection;
 import java.util.Optional;
 
 @Repository
 public interface RFIDLinkRepository extends JpaRepository<RFIDLink, Long> {
     Optional<RFIDLink> findByRfid(String rfid);
 
     Optional<RFIDLink> findByTicketId(Long ticketId);
+
+    @Query("select rf from RFIDLink rf where rf.ticket.owner.email = :email")
```

Comment:
Cheats. You can do this with proper method naming

Reply:
I tried that option at first, but I couldn't find how. If you provide a source with explanation I will do it with method naming.

Reply:
I can help you with this on Monday morning at CH after my lecture around 11 ish, if you want!

Reply:
I found out how to do it. It is implemented!